v-2.0.17: add exclude empty zero forms flag to locales download endpoint

paulbaum opened the pull request:
This adds the flag "exclude empty zero forms" to the locale#download endpoint

ffittschen commented:
@paulbaum @egze @theSoenke Hi, due to the [latest `phrase-cli` tag](https://github.com/phrase/phrase-cli/releases/tag/v2.0.17) having a `v` prefix, the urls created by [this line](https://github.com/phrase/homebrew-brewed/blob/master/Formula/phrase.rb#L17) are not correct and result in a `404`:

https://github.com/phrase/phrase-cli/releases/download/v2.0.17/phrase_macosx_amd64.tar.gz

Can you please update this tap to fix the URLs?

paulbaum commented:
Hi @ffittschen , thanks for the quick heads up! I am sorry for the inconvenience! We renamed the tag now and removed the v prefix. Does it work for you again?

ffittschen commented:
Hi @paulbaum, thanks for the quick fix. Installations with homebrew are working again.
